Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -1052,6 +1052,15 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     if shouldSkip(node) {
       return .skipChildren
     }
+
+    if node.fixity.presence == .missing {
+      addDiagnostic(
+        node.fixity,
+        .missingModifierInOperatorDeclaration,
+        handledNodes: [node.fixity.id]
```

Comment:
I think that's ok. Do we need to add all possible Fix-Its on this type of missing node in the future? e.g. the mandatory `_version` or `_underlyingVersion` label of the second parameter of `canImport`.

Reply:
> Before this, my thought was not to add the Fix-Its since we could not infer what the user miss here.

As long as there is a reasonably short list of possible options, I think offering Fix-Its is just a nice thing to do. And we don’t need to know what the user wanted to write because there will be three different Fix-Its, one for each fixity, and the user can choose which one they want to apply.

And yes, I agree that offering the same for `_version` and `_underlyingVersion` would also be nice.